codelist creation.md

```diff
@@ -60,13 +60,11 @@ Once a draft codelist has been agreed, it must be signed-off by a "data expert"
 
 ## Add to [codelists.opensafely.org](https://codelists.opensafely.org)
 
-* Go to the openSAFELY [codelist admin page](https://codelists.opensafely.org/admin). You will need an editor account.
-* Under "CODELISTS", click "Add".
+* Go to the openSAFELY [new codelist page](https://codelists.opensafely.org/codelist/opensafely/). You will need an editor account.
```

Comment:
"Ask one of the tech team for one if you do not have one."

Reply:
Fixed in a944b80